indexer/bin/indexer.js

```diff
@@ -54,18 +54,34 @@ export default {
       console.log(
         `New proof set (set_id=${payload.set_id}, owner=${payload.owner})`,
       )
-      await env.DB.prepare(
-        `
+      try {
+        await env.DB.prepare(
```

Comment:
I suggest that that we leave the `ON CONFLICT` statement but introduce a try / catch block for capturing errors that might occure during the insert.